forum/views.py

```diff
@@ -19,6 +19,11 @@
 )
 
 
+# class UserProfileView(DetailView):
+#     model = request.user
+#     pk_url_kwarg = 'user_id'
+    #  Not sure how to add user
```

Comment:
Uncomment this (it works).

To add a User, do it in the Django admin for now. Once you add `django-registration` or `django-allauth`, you'll be to add Users the normal way.

Reply:
done.  starting reading a bit on the django-registration.  seems like it might be the lighter light for right now, but the django-allauth seems like a powerful tool to have exposure too